src/referencePackages/src/system.runtime.compilerservices.unsafe/6.1.1/System.Runtime.CompilerServices.Unsafe.6.1.1.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project will need to be added to https://github.com/dotnet/source-build-reference-packages/blob/main/eng/Build.props#L45 per the [instructions](https://github.com/dotnet/source-build-reference-packages/blob/main/README.md#workflow)

```
Add DependencyPackageProjects for all new projects in the [eng/Build.props](https://github.com/dotnet/source-build-reference-packages/blob/main/eng/Build.props#L9) in the correct dependency order.
```